addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -99,7 +99,7 @@
         </RappresentanteFiscale>
         <CessionarioCommittente>
             <DatiAnagrafici>
-                <IdFiscaleIVA>
+                <IdFiscaleIVA t-if="buyer_info['vat']">
```

Comment:
Is there a case where we could have a `vat` and not a `country_code`?
I haven't looked everywhere, I know it should be prevented by `_l10n_it_edi_check_invoice_configuration`, but the method rendering this template is `invoice_generate_xml` which is open.